build/parser.mjs

```diff
@@ -968,9 +976,8 @@ class Parser {
     if (target.EstimatedVaultedDate) {
       item.estimatedVaultDate = this.normalizeOggDate(target.EstimatedVaultedDate);
     }
-    if (target.Vaulted) {
-      item.vaulted = target.Vaulted;
-    }
+
+    item.vaulted = wikiaItem.vaulted || target.vaulted;
```

Comment:
is the target.vaulted that unreliable so we need to depend directly from the wikia if available?

Reply:
if you want we could use ogg as the primary and the wikia second

Reply:
nah i'm ok having the wikia first

Reply:
i think i was saying it in a weird way that wasn't helpful. if ogg is defined, use ogg. if wiki is defined and ogg is not, use wiki... imo.

the fallback language, i think, was making my meaning unclear

Reply:
so no more